ikarus/finiteelements/mechanics/linearelastic.hh

```diff
@@ -158,32 +147,61 @@ public:
   [[nodiscard]] size_t numberOfNodes() const { return numberOfNodes_; }
   [[nodiscard]] int order() const { return order_; }
 
-  /**
-   * \brief Calculates the scalar energy for the given FERequirementType.
-   *
-   * \param par The FERequirementType object.
-   * \return The scalar energy.
-   */
-  inline double calculateScalar(const FERequirementType& par) const { return calculateScalarImpl<double>(par); }
+  using SupportedResultTypes = std::tuple<decltype(makeRT<ResultTypes::linearStress>())>;
```

Comment:
Why do we create a Dummy to then use its type?

Reply:
I wanted to hide that we artificially inject doube,1,1 therein.

Reply:
Yeah thats a problem with this approach, we / i maybe have to think of something nicer

Reply:
Maybe we can do something like:

```cpp
  struct resultTypeName                                                                                         
  {                                                                                                             
    friend auto toString(resultTypeName) { return #resultTypeName; }                                            
    
    template <typename ScalarType, int gridDim, int worldDim>                                                            
    using type       = Eigen::Matrix<ScalarType, gridDim*gridDim*something, bla, 0, blub*worldDim, Dynamic>;              
    using Vectorizer = VectorizeStruct;                                                                         
    using Matricizer = MatricizeStruct;                                                                         
                                                                                                                
    template <typename ScalarType_, int gridDim_, int worldDim_>                                                
    using Rebind = resultTypeName<ScalarType_, gridDim_, worldDim_>;                                            
  }
```

The we have a non-template again.

Reply:
and maybe hardcode the ScalarType to double.

Reply:
This can be tackled in another PR 
- [ ] Refactor ResultTypes to be no tempaltes